lumigator/python/mzai/backend/lm_buddy/cli/finetune.py

```diff
@@ -0,0 +1,13 @@
+import click
```

Comment:
Do we want to trim some things out like finetuning?

Reply:
Yup, that will be step 2 described in the PR. I want to do this direct port first, test to make sure functionality is basically the same, and then the second PR following this will be removing: finetuning, ragas and prometheus if we decide we don't need them anymore, and perhaps some of the other pieces.